Makefile.am

```diff
@@ -68,6 +68,11 @@ mancheck:
 			xargs mandoc -Tlint -Werror; \
 	fi
 
+testscheck:
+	@find ${top_srcdir}/tests/zfs-tests/tests -type f -name '*.ksh' \
```

Comment:
Should we test other known file extensions for executable correctness?  Maybe we could add a check for files with the `.kshlib` extension as well (most don't have executable permission, some do)? And should this check that files with extensions such as `.cfg` aren't executable? Thoughts?

Reply:
Good idea, extending this to verify both `.kshlib` and `.cfg` files aren't executable would be good.

Reply:
Agreed, let me test this locally and update the PR.